setup.py

```diff
@@ -66,6 +68,7 @@ def find_version(*file_paths):
     namespace_packages=["luma"],
     zip_safe=False,
     install_requires=["luma.core>=1.16.1"],
+
```

Comment:
no need for newline

Reply:
and probably needs to rely on luma.core >= 1.16.2

Reply:
Removed newline